src/shared/Atlassian.Bitbucket/OAuth2ClientRegistry.cs

```diff
@@ -30,15 +27,20 @@ public BitbucketOAuth2Client Get(InputArguments input)
             return CloudClient;
         }
 
-        public void Dispose()
+        protected override void ReleaseManagedResources()
         {
-            http.Dispose();
-            settings.Dispose();
-            cloudClient = null;
-            dataCenterClient = null;
+            _httpClient.Dispose();
```

Comment:
Will this cause an exception if `_httpClient` is `null`?

Reply:
Yes! Thanks for catching. Fixed in next push.